app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -18,11 +18,11 @@
 
     <!--[if lt IE 9]><%= javascript_include_tag "ie.js", integrity: true, crossorigin: "anonymous" %><![endif]-->
 
-    <%# the colour used for theme-color is the standard palette $black from
-        https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/colours/_palette.scss %>
+    <!-- the colour used for theme-color is the standard palette $black from GOVUK Frontend -->
```

Comment:
**Reviewing commit by commit**

This line is now a HTML comment, which means it will appear in the markup of the page. I think this and line 25 should go back to being rails comments, we don't want to add any page weight we don't need.